app/views/allocations/_female_complexity_level.html.erb

```diff
@@ -0,0 +1,9 @@
+<tr class="govuk-table__row">
+  <td class="govuk-table__cell">Complexity level</td>
+  <td id=complexity-level class="govuk-table__cell table_cell__left_align">
+    <%= @prisoner.complexity_level.capitalize %>
+    <% unless auto_delius_import_enabled?(@prison.code) %>
+      <a class="govuk-link pull-right" href="<%= edit_prison_prisoner_complexity_level_path(@prison.code, @prisoner.offender_no) %>">Change</a>
+    <% end %>
```

Comment:
This flag is a piece of legacy that we should be ripping out. There's no reason that I ca think of why we should not display this change link if this switch is set.